packages/apidom-ls/src/config/openapi/openapi3_1/lint/jsonSchemaDialect--format-uri.ts

```diff
@@ -0,0 +1,15 @@
+import ApilintCodes from '../../../codes';
```

Comment:
Name of the file should be `json-schema-dialect--format-uri.ts` (to be consistent with the rest of the codebase).